scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE,
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE,
+  insurance CHARACTER VARYING
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  patient_name CHARACTER VARYING,
```

Comment:
These should be `patient_id` and `doctor_id`, references to their respective tables.